autopts/wid/bass.py

```diff
@@ -214,6 +216,28 @@ def hdl_wid_116(_: WIDParams):
     Please expose a Broadcast Receive State with BIG_Encryption
     field indicating Broadcast_Code required.
     """
+
+    stack = get_stack()
+    broadcaster_addr_type = 0x00  # Public address
+    broadcaster_addr = '001122334455'
+    advertiser_sid = 0x01
+    broadcast_id = 0x123456
+
+    bis_sync = 0
+    metadata_len = 0
+    subgroups = struct.pack('<IB', bis_sync, metadata_len)
```

Comment:
Suggest to add a comment stating why these values are set. 

For the specific test case you are working on, any values work as we just need to expose the BIG Sync state to be 0x01 (which oddly enough is the only state you aren't setting in a variable here). 

It looks like 116 is only called for `BASS/SR/CP/BV-21-C`, so I don't expect this code to break any other tests

Reply:
@omkar3141 looks like you removed the comment again. Please re-add the comment about the values just need to be valid values, and that they do not need to correspond to any actual broadcast source. The only field, as per the WID description, that needs a specific value, is the BIG_Encryption field